### DISCUSSION TITLE
Reinforced Learning methods and Platform.

### DISCUSSION THREAD

**Discussion by SalikSayyed** [SalikSayyed/Smartify#10] [Show and tell]
READ ABOUT EXACTLY HOW TO IMPLEMENT REINFORCED LEARNING INTO NLP.
LEARNING ABOUT NLP LIBRARIES.

**Comment by Rutik20** [comment]
NLP Libraries

-  NLTK
- Spacy
- TextBlob
- Gensim
- Core NLP

More about NLP Libraries-
[https://www.softkraft.co/python-nlp-libraries-features-us-cases-pros-and-cons/](url)

[https://sunscrapers.com/blog/8-best-python-natural-language-processing-nlp-libraries/](url)

**Comment by SalikSayyed** [comment]
> NLP Libraries
> 
> * NLTK
> * Spacy
> * TextBlob
> * Gensim
> * Core NLP
> 
> More about NLP Libraries-
> [https://www.softkraft.co/python-nlp-libraries-features-us-cases-pros-and-cons/](url)
> 
> [https://sunscrapers.com/blog/8-best-python-natural-language-processing-nlp-libraries/](url)

Keep looking.. and test, select one which best fits our need as per your view.

**Comment by Rahul214-Coder** [comment]
I am working on TextBlob library but I can't run the program on my laptop I am getting some difficulties .So  anyone can run this program by replacing text file with Epub /pdf file. I am sharing the youtube link in which there are many function of Textblob which will help in this project .

I am sharing some screenshot of function.
![Screenshot_2021-04-04-20-07-15-25](https://user-images.githubusercontent.com/72076722/113519990-9795f480-95ad-11eb-886e-e28d294e4d9a.jpg)
![Screenshot_2021-04-04-20-08-44-22](https://user-images.githubusercontent.com/72076722/113519991-9a90e500-95ad-11eb-98e5-54f30411a5b3.jpg)
![Screenshot_2021-04-04-20-22-09-63](https://user-images.githubusercontent.com/72076722/113519992-9ebd0280-95ad-11eb-8080-728b2ff87878.jpg)
![Screenshot_2021-04-04-20-22-39-94](https://user-images.githubusercontent.com/72076722/113519993-a11f5c80-95ad-11eb-97b7-c620b29e0a32.jpg)
![Screenshot_2021-04-04-20-24-08-54](https://user-images.githubusercontent.com/72076722/113519996-a381b680-95ad-11eb-8ba7-709f0bd59ac2.jpg)
![Screenshot_2021-04-04-20-24-37-64](https://user-images.githubusercontent.com/72076722/113519998-a54b7a00-95ad-11eb-95e6-216bf0fd3354.jpg)
![Screenshot_2021-04-04-20-25-12-46](https://user-images.githubusercontent.com/72076722/113520002-a9779780-95ad-11eb-97ff-4e74037b142c.jpg)
![Screenshot_2021-04-04-20-25-14-22](https://user-images.githubusercontent.com/72076722/113520005-ada3b500-95ad-11eb-9cb5-70bc6dfe8bd1.jpg)

By seeing this I found that we should use

1.blob.translate
2.ngrams
3.blob.sentences using If statement

for  more you also go through this


[https://www.youtube.com/watch?v=Cxzyalrjr94&list=PL_92WMXSLe_-RkWW5zAQZ-gMdVqZ7T-_F](url)

link of textblob documentation
https://textblob.readthedocs.io/en/dev/#:~:text=TextBlob%20is%20a%20Python%20(2,classification%2C%20translation%2C%20and%20more.

**Reply by SalikSayyed** [reply]
Were u able to implement it? Try that.
